datasets/gov/vstt/saugomu_ter_kadastras.csv

```diff
@@ -0,0 +1,1105 @@
+,,,,,id_vart,integer,,,,2,open,,Abonento klasifikatorius,
+,,,,,,,,,,,,,,
+,,,,BiosferosPoligonas,,,id,,,4,,,Biosferos poligonai,
+,,,,,id,string,,,,4,open,,Identifikavimo kodas,
+,,,,,pavadinimas,string,,,,4,open,,Biosferos poligono pavadinimas,
+,,,,,gkodas,string,,,,2,open,,InGIS geokodas,Saugomos teritorijos tipas
+,,,,,plotas,number,ha,,,4,open,,Plotas, 
+,,,,,steigejas,string,,,,2,open,,Steigėjas – institucija,
+,,,,,steigimas,date,D,,,4,open,,Steigimo data,
+,,,,,steig_ta,string,,,,4,open,,Įsteigimo teisės aktas,
+,,,,,steig_tikslas,string,,,,4,open,,Steigimo tikslas,Aprašomoji informacija
+,,,,,bendr_reglam,integer,,,,4,open,,Reglamentas,Bendrieji veiklos reglamentai nuoroda į teisės aktą
+,,,,,,enum,,,1,,,,Rezervatai,
+,,,,,,,,,2,,,,Draustiniai,
+,,,,,,,,,3,,,,Parkai,
+,,,,,,,,,4,,,,Sklypai,
+,,,,,,,,,5,,,,"Biosferos Poligonai, Biosferos Rezervatai",
+,,,,,,,,,6,,,,Gamtos paveldo objektai,
+,,,,,spec_reglam,string,,,,4,open,,Reglamentai,Specialieji / individualieji veiklos reglamentai
+,,,,,saltiniai,string,,,,2,open,,Duomenų  šaltiniai,Nurodomi moksliniai tyrimai ir publikacijos
+,,,,,tarpt_svarba_status,integer,,,,4,open,,Teritorijos statusas,Saugomai teritorijai arba jos daliai suteiktas tarptautinės svarbos  (įskaitant Europos Bendrijų svarbos) teritorijos statusas
+,,,,,,enum,,,0,,,,Nėra duomenų,
+,,,,,,,,,1,,,,Unesco,
+,,,,,,,,,2,,,,Ramsar,
+,,,,,,,,,3,,,,Buveinių apsaugos,
+,,,,,,,,,4,,,,Paukščių apsaugos,
+,,,,,,,,,5,,,,Helcom,
+,,,,,,,,,6,,,,Paukščių ir buveinių apsaugos,
+,,,,,tarpt_svarba_apras,string,,,,2,open,,Aprašas,Saugomai teritorijai arba jos daliai suteiktas tarptautinės svarbos  (įskaitant Europos Bendrijų svarbos) teritorijos statuso aprašymas
+,,,,,duomen_monitor,string,,,,2,open,,Monitoringas,Aplinkos stebėsenos (monitoringo) duomenys
+,,,,,pastabos,string,,,,4,open,,Pastabos,
+,,,,,reiksme,integer,,,,2,open,,Objekto reikšmė,Gamtos paveldo objekto reikšmė: valstybės saugomi savivaldybės saugomi
+,,,,,steigejas_galiojantis,string,,,,2,open,,Steigėjas,Institucija kurios potvarkiu šiuo metu galioja saugoma teritorija arba jos riba
+,,,,,galiojantis,date,D,,,4,open,,Steigimo data,Šiuo metu galiojančio saugomos teritorijos steigimo teisės akto data
+,,,,,steig_ta_galiojantis,string,,,,4,open,,Teisinis statusas,Teisės aktas pagal kurį šiuo metu galioja saugomos teritorijos riba reglamentai arba šis teisės aktas yra šiuo metu galiojantis saugomos teritorijos teisinis pagrindas
+,,,,,tvarkymo_zona,string,,,,4,open,,Zona,Duomenys apie kraštovaizdžio tvarkymo zonas
+,,,,,funkc_prior_zon,string,,,,4,open,,Prioriteto zona,Duomenys apie biosferos poligono funkcinio prioriteto zonas
+,,,,,planavimo_dok,string,,,,4,open,,Planavimo dokumentas,Duomenys apie parengtus planavimo dokumentus
+,,,,,vieta,string,,,,2,open,,Vieta,Aprašomoji informacija (savivaldybė seniūnija jei miškas – papildomai miškų urėdija girininkija miško kvartalas ir sklypas)
+,,,,,geometrija,geometry,,,,3,open,,Ploto geometrija,
```

Comment:
```suggestion
,,,,,geometrija,geometry(3346),,,,3,open,,Ploto geometrija,
```
Duomenyse nurodytas SRID, reikėtų jį surodyti ir ADSA.
Taip pat prašome įsitikinti, kad duomenyse erdviniai duomenys pateikti tinkamai:
![image](https://user-images.githubusercontent.com/85924252/217271861-75da8c30-1051-44f2-80a8-c0d03f6a2a26.png).

Šis pastebėjimas jau buvo teiktas anksčiau:
jeigu yra nurodytas SRID kodas, o duomenų tipai skiriasi, siūlome žymėti 4 brandos lygiu.